src/services/kratos/index.ts

```diff
@@ -1 +1,50 @@
+import { AuthenticationKratosError, UnknownKratosError } from "./errors"
```

Comment:
why is this file not using an interface?

Reply:
I'm still working on the interface as I learn about how kratos works